hack/update-compatibility.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mapfile -t apiPkgs < <(go list -mod=vendor  -find  ./... | grep -v -E "$(go list)/(pkg|cmd|tools|hack)" | grep -E "$(go list)/[^/]+/[^/]+")
+inputArg=$(printf ",%s" "${apiPkgs[@]}")
+inputArg=${inputArg:1}
+
+go run github.com/openshift/api/cmd/openshift-compatibility-gen --input-dirs "$inputArg"
```

Comment:
is this always running the local copy? Would have expected `go run ./cmd/....`

Reply:
I remember researching this and as long as I don't add an `@version` it should be using the local version. Unfortunately, I can't find supporting documentation for that at the moment.

Reply:
This is working as expected and running the local copy.